Controller/ProfilerController.php

```diff
@@ -88,7 +88,9 @@ private function explainSQLServerPlatform(Connection $connection, $query)
 
     private function explainOtherPlatform(Connection $connection, $query)
     {
-        return $connection->executeQuery('EXPLAIN '.$query['sql'], $query['params'], $query['types'])
+        $params = $query['params'] instanceof Data ? $query['params']->getValue(true) : $query['params'];
```

Comment:
What is `Data`? There is no import for this class and Doctrine\Bundle\DoctrineBundle\Controller\Data doesn't exist either - probably missing import?

Reply:
I've missed to add the import. It's done now